mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/SmartTestingMavenConfigurer.java

```diff
@@ -86,9 +91,18 @@ private void calculateChanges() {
         changeStorage.store(changes);
     }
 
-    private void configureExtension(MavenSession session, Configuration configuration) {
+    private void configureExtensionPrepareReports(MavenSession session, Configuration configuration) {
```

Comment:
Can we give better name? This name is ambiguous

Reply:
What would you suggest @dipak-pawar ?

Reply:
I think it can stay like it was before. It's clear enough from the content what is going on.

Reply:
Ok, if you're fine with the original name `configureExtension` also for copying the reports, why not. 
For me, configure extension was related to modifying pom.xml files and similar, not preparing reports - this is why I changed the name. But I agree that the name is weird - this is always one of my weaknesses :-D.

Reply:
I think it was ambiguous already. But that is fine - this method takes care of configuring extension and within it we have detailed steps explaining what really is the "configuration" all about.